modules/tomlplusplus/3.4.0/patches/bazel.patch

```diff
@@ -0,0 +1,81 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 0000000..4546e55
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,62 @@
++load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
++
++exports_files(["toml.hpp"])
++
++cc_library(
++    name = "toml++",
```

Comment:
Please add a `tomlplusplus` alias so that users can use the usual shortcut.

Reply:
done